Documentation/Administration/Installation/ClassicMode/Docker.rst

```diff
@@ -0,0 +1,316 @@
+    and a MariaDB container.
+
+    For production or team development, use
+    `Composer-based <https://docs.typo3.org/permalink/t3coreapi:installation-composer>`_ TYPO3,
+    `Version control (Git) <https://docs.typo3.org/permalink/t3coreapi:version-control>`_
+    and a custom Docker setup.
+
+..  contents::
+
+..  _classic-docker-quickstart:
+
+Quick start
+===========
+
+To quickly launch TYPO3 in classic mode with Docker:
+
+..  code-block:: bash
+    :caption: ~/projects/typo3demo/$
+
+    mkdir -p fileadmin typo3conf typo3temp
+
+    # Ensure TYPO3 can write to these directories
+    chmod -R 777 fileadmin typo3conf typo3temp
+
+    docker network create typo3-demo-net
+
+..  literalinclude:: _codesnippets/_DockerDbDemo.sh
+    :language: bash
+    :caption: ~/projects/typo3demo/$
+
+..  literalinclude:: _codesnippets/_DockerRunTypo3Demo.sh
+    :language: bash
+    :caption: ~/projects/typo3demo/$
+
+Then open:
+
+..  code-block:: text
+
+    http://localhost:8080
+
+Use these database settings in the TYPO3 installer:
+
+-   **Database Host**: `typo3db`
+-   **Username**: `db`
+-   **Password**: `db`
+-   **Database Name**: `db`
+
+..  _classic-docker-prerequisites:
+
+Prerequisites for using TYPO3 in Docker locally
+===============================================
+
+-   Docker installed. See
+    `https://docs.docker.com/get-docker/ <https://docs.docker.com/get-docker/>`_.
+-   Basic knowledge of Docker.
+-   A web browser to access TYPO3.
+
+..  _classic-docker-setup:
+
+Step-by-step Docker setup
+=========================
+
+..  _classic-docker-project:
+
+1. Prepare a project directory
+------------------------------
+
+Create a local project directory and subfolders for TYPO3's writable directories:
+
+..  literalinclude:: _codesnippets/_DockerDirDemo.sh
+    :language: bash
+    :caption: ~/projects/$
+
+..  _classic-docker-network:
+
+2. Create a user-defined Docker network
+---------------------------------------
+
+..  code-block:: bash
+    :caption: ~/projects/typo3demo/$
+
+    docker network create typo3-demo-net
+
+..  _classic-docker-db:
+
+3. Start the MariaDB database container
+---------------------------------------
+
+..  literalinclude:: _codesnippets/_DockerDbDemo.sh
+    :language: bash
+    :caption: ~/projects/typo3demo/$
+
+..  _classic-docker-typo3:
+
+4. Start the TYPO3 container with mounted writable directories
+--------------------------------------------------------------
+
+..  literalinclude:: _codesnippets/_DockerRunTypo3Demo.sh
+    :language: bash
+    :caption: ~/projects/typo3demo/$
+
+..  important::
+    **Ensure folder permissions are correct**
+
+    Mounted folders must be writable by the web server inside the container.
+    If you use mounted folders, adjust their permissions on your host system:
+
+    ..  code-block:: bash
+        :caption: ~/projects/typo3demo/$
+
+        chmod -R 777 fileadmin typo3conf typo3temp uploads
```

Comment:
This is not recommended for folders. I understand this is the easiest way to solve this so you don't have to fiddle with users and user id. in the container. It might be good to add a warning that this should be avoided to use `777`. 

This is only needed for linux or wsl.

Reply:
Ok, added a section on how to deal with permissions on different OS, commented it out, left a warning about only to do during development